File: python_draft/src/javascript/type3/findMaxExtra.js
function findMaxV2(values) {
  if (!values || values.length === 0) return null;
  let currentMax = values[0];
  for (const number of values) {
    if (number > currentMax) currentMax = number;
  }
  console.log("Max found:"); // extra
  return currentMax;
}
module.exports = { findMaxV2 };

File: python_draft/src/cpp/type3/find_max_extra.cpp
#include <vector>
#include <iostream>
int findMaxV2(const std::vector<int>& values){
    if(values.empty()) return -2147483648;
    int currentMax = values[0];
    for(int number : values){
        if(number > currentMax) currentMax = number;
    }
    std::cout << "Max found:" << std::endl;
    return currentMax;
}
